tests/TestCase.php

```diff
@@ -331,6 +332,7 @@ protected function getFakeValue($attribute, $class)
                             sprintf('Cannot generate fake value for "%s :: %s"', $class, $attribute)
                         );
                 }
+                // no break
```

Comment:
So what is the correct indent 😄  ?

Reply:
Weird that cs-fix don't detect the others. Hangon. I'll just revert those chunks to not clutter the PR. 😅